css/style.css

```diff
@@ -22,11 +22,12 @@ section.landing {
   margin:0;
   padding:0;
   background-size: cover;
-  background-color:rgba(0,46,102,.8);
+  //background-color:rgba(0,46,102,.8);
```

Comment:
Instead of commenting it out (also I don't think CSS is commented this way), let's remove this line of code.